lib/Comment.php

```diff
@@ -29,7 +29,7 @@
  * <p class="comment-attribution">- Sullivan Ballou</p>
  * ```
  */
-class Comment extends Core implements CoreInterface, MetaInterface {
+final class Comment extends Core implements CoreInterface, MetaInterface {
```

Comment:
We won’t make classes `final`. Extending Timber’s classes for WordPress objects is one of the [basic concepts of working with Timber](https://github.com/timber/timber/blob/2.x/docs/guides/class-maps.md).

Reply:
Oh! I see!

Reply:
Has been force-removed.